examples/hcl2/policy/deny_test.rego

```diff
@@ -17,9 +17,15 @@ test_correctly_encrypted_azure_disk {
 }
 
 test_unencrypted_azure_disk {
-	deny["Azure disk `sample` is not encrypted"] with input as {"resource": {"azurerm_managed_disk": {"sample": {"encryption_settings": {"enabled": false}}}}}
+	cfg := parse_config_file("unencrypted_azure_disk.tf")
+	deny["Azure disk `sample` is not encrypted"] with input as cfg
 }
 
 test_fails_with_http_alb {
```

Comment:
I couldn't see a ref for `conftest verify -p examples/hcl2/policy` in any of the acceptance test files, so it might be worth adding it if I'm not missing a point

btw, nice feature :+1:

Reply:
Good catch! I incorrectly assumed everything in `examples/` was tested. Added.